requirements/scenarios.md

```diff
@@ -1,21 +1,21 @@
-# Notary v2 Signing Scenarios
+# Notary Project Signing Scenarios
 
 As containers and cloud native artifacts become the common unit of deployment, users want to know the artifacts in their environments are authentic and unmodified.
 
-These Notary v2 scenarios define end-to-end scenarios for signing artifacts in a generalized way, storing and moving them between OCI compliant registries, validating them with various artifact hosts and tooling.
-Notary v2 focuses on the signing of content, enabling e2e workflows, without specifying what those workflows must be.
+These Notary Project scenarios define end-to-end scenarios for signing artifacts in a generalized way, storing and moving them between OCI compliant registries, validating them with various artifact hosts and tooling.
```

Comment:
It seems that `Notary Project` becomes a special term as `P` is in upper case. If so, will it be grammatically correct to say `The Notary Project`?